src/ray/gcs/gcs_server/gcs_placement_group_manager.cc

```diff
@@ -461,26 +459,37 @@ void GcsPlacementGroupManager::HandleWaitPlacementGroupUntilReady(
   RAY_LOG(DEBUG) << "Waiting for placement group until ready, placement group id = "
                  << placement_group_id;
 
-  auto callback = [placement_group_id, reply, send_reply_callback](const Status &status) {
-    RAY_LOG(DEBUG)
-        << "Finished waiting for placement group until ready, placement group id = "
-        << placement_group_id;
+  WaitPlacementGroup(placement_group_id, [reply, send_reply_callback,
+                                          placement_group_id](Status status) {
+    if (status.ok()) {
+      RAY_LOG(INFO)
```

Comment:
Why change it to INFO?

Reply:
It should be `DEBUG `, I will update this.